.github/workflows/release.yml

```
@@ -0,0 +1,30 @@
+name: gh-pages
+on:
+  release:
+    types: [published, edited]
+
+concurrency:
+  group: gh-pages
+
+jobs:
+  update-helm-repo:
+    name: Update gh-pages helm repo index
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install helm
+      run: |
+        curl -sfL https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash -s -- --version v3.12.3
+
```

```suggestion
    - name: Install Helm
      uses: azure/setup-helm@v3
```
I would suggest to use action image here instead which is doing the same thing.

---

Thanks, makes sense, fixed